gnomad_qc/v3/annotations/generate_sv_hists.py

```diff
@@ -0,0 +1,197 @@
+# noqa: D100
+import argparse
+import logging
+
+import hail as hl
+from gnomad.utils.slack import slack_notifications
+
+from gnomad_qc.slack_creds import slack_token
+from gnomad_qc.v3.resources.annotations import sv_age_and_gq_hists
+from gnomad_qc.v3.resources.basics import (
+    gnomad_sv_autosome_vcf_paths,
+    gnomad_sv_release_samples_list_path,
+    gnomad_sv_sex_vcf_paths,
+    temp_gnomad_sv_mt_path,
+)
+from gnomad_qc.v3.resources.meta import meta
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s: %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger("generate_gnomad_sv_histograms")
+logger.setLevel(logging.INFO)
+
+
+def generate_hists(mt: hl.MatrixTable) -> hl.Table:
+    """
+    Generate histograms for age and GQ for gnomAD SVs.
+
+    :param mt: gnomAD SV MatrixTable.
+    :return: Table with histograms.
+    """
+    logger.info("Generating histograms for age and GQ...")
+    hists = mt.select_rows(
+        age_hist_het=hl.or_missing(
+            ~mt.filters.contains("MULTIALLELIC"),
+            hl.agg.filter(mt.GT.is_het(), hl.agg.hist(mt.age, 30, 80, 10)),
```

Comment:
if we move this to gnomad_methods, these numbers should be params (like in the `age_hists_expr` function https://github.com/broadinstitute/gnomad_methods/blob/main/gnomad/utils/annotations.py)